ci/it/configs/quesma-ingest.yml.template

```diff
@@ -97,7 +97,7 @@ processors:
             - my-clickhouse-instance
         "*":
           target:
-            - my-minimal-elasticsearch
+            - my-clickhouse-instance
```

Comment:
Will this not change the default behavior?

Reply:
I've moved this test to the separate test case. It has its own config now.